modules/ROOT/pages/install-helm.adoc

```diff
@@ -0,0 +1,169 @@
+= Installing Runtime Fabric on Self-Managed Kubernetes Using Helm
+
+[IMPORTANT]
+====
+Installing Runtime Fabric on Self-Managed Kubernetes requires tasks that must be performed by both IT administrators and MuleSoft organization administrators.
+====
+
+You can use Helm to install Anypoint Runtime Fabric on an Amazon Elastic Kubernetes Service (Amazon EKS), Azure Kubernetes Service (AKS) or Google Kubernetes Engine (GKE) installation that you manage. 
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environment, ensure:
+
+* You have reviewed the architecture and requirements outlined in xref:index-self-managed.adoc[Runtime Fabric on Self-Managed Kubernetes]
+* You have xref:install-self-managed-network-configuration.adoc[configured your network to support Runtime Fabric on Self-Managed Kubernetes]
+* You have installed and configured your Kubernetes environment as follows:
++
+** Running an EKS, AKS, or GKE Kubernetes environment. Other Kubernetes environments are not supported.
+** Running a xref:index-self-managed.adoc#supported-kubernetes-versions[supported Kubernetes version]
+** Running an ingress controller to send external requests to applications.
+* You have installed Helm 3 or later.
+
+[NOTE]
+--
+If you use Helm to manage your Runtime Fabric installation, you can still later use the `rtfctl` command line utility to manage Mule license keys and proxies. However, you’ll need to track any post-installation changes you make, as explained in the upgrade documentation [add link!!!]. 
+--
+
+== Create a Runtime Fabric Using Runtime Manager
+
+To install Runtime Fabric on Self-Managed Kubernetes with Helm, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
+
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+* Google Kubernetes Engine
+. Review the Support responsibility disclaimer, and if you agree, click *Accept*.
+. Click Helm.
+. Note that your activation data is captured in the code example provided by Runtime Manager. You will need this activation data to install Runtime Fabric.
```

Comment:
This shouldn't be a bullet because it isn't worded as a step. You could say something like this to keep it a step: 
Save the activation data captured in the code example provided by Runtime Manager...

Reply:
good call